### PR TITLE
Feature: Add separateDialCode option

### PR DESCRIPTION
**Pull request by AleksandrBorovkov**

### CODE REVIEW

**Comment by dcyriller** [comment]
You make this new property as an opt-in 👍.

May you add a test for this new option in the file `tests/integration/components/phone-input-test.js`?

Also, it would be nice to add a section in the playground to demonstrate this option that you add. You could add at the bottom of [this file](https://github.com/qonto/ember-phone-input/blob/v2.0.7/tests/dummy/app/pods/docs/components/phone-input/all-options/template.hbs) a h2 title: `separateDialOption` with the following snippet:
```hbs
<h2>separateDialOption</h2>
{{#docs-demo as |demo|}}
  {{#demo.example name="phone-input-separate-dial-option.hbs"}}

    <p>{{phone-input separateDialCode=true initialCountry="us" number=separateDialNumber update=(action "updateSeparateDialOption")}}</p>

  {{/demo.example}}

  {{demo.snippet "phone-input-separate-dial-option.hbs"}}
{{/docs-demo}}
```

Then you would add at the top of the file: `<h2>Basic Options</h2>`.

Could add this two things?

**Comment by AleksandrBorovkov** [comment]
@dcyriller Added

**Comment by AleksandrBorovkov** [comment]
@dcyriller Hello, When are you planning to merge my PR ? I need this feature for my project

File: tests/dummy/app/pods/docs/components/phone-input/all-options/template.hbs [hunk 1]
```diff
@@ -17,3 +18,16 @@
 
   {{demo.snippet "phone-input-all-options.hbs"}}
 {{/docs-demo}}
+
+<h2>separateDialOption</h2>
```
Comment (dcyriller) [inline tests/dummy/app/pods/docs/components/phone-input/all-options/template.hbs]:
```suggestion
<h2>`separateDialCode` option</h2>
```

File: tests/integration/components/phone-input-test.js [hunk 2]
```diff
@@ -39,6 +39,31 @@ module('Integration | Component | phone-input', function(hooks) {
...
   })
 
+  test('renders the value with separate dial code option', async function(assert) {
+    assert.expect(3)
+
+    await this.owner.lookup('service:phone-input').load()
+
+    const newValue = '2'
+    this.set('separateDialNumber', null)
+    this.set('update', () => {})
```
Comment (dcyriller) [inline tests/integration/components/phone-input-test.js]: Do you mind simplifying the test a bit? 

Here you could set the update function to:
```
    this.set('update', value => {
      this.set('separateDialNumber', value)
    })
```
to not change it afterwards. I feel like modyfing it in the test scope makes things harder to understand.

**Comment by AleksandrBorovkov** [comment]
@dcyriller Agree. I've made fix.

**Comment by dcyriller** [comment]
Released as v3.0.0 (I had to bump the major version due to a breaking PR not released: node 6 support drop).

**Pull request merged**